### DISCUSSION TITLE
CentOS7 Content

### DISCUSSION THREAD

**Discussion by EngThis** [ComplianceAsCode/content#6568] [Q&A] [answered]
After building the CentOS content I noticed the ssg-centos7-xccdf.xml does not have a stig profile for the DISA STIG. Is there a way I can build it using your tool?

**Comment by ggbecker** [comment, marked as answer]
The CentOS content is derived from the RHEL one but any profile developed for RHEL is not carried to CentOS other than the one defined in https://github.com/ComplianceAsCode/content/blob/543a04f6f60cc66cc7a5fe470dae052a4d6c8762/ssg/constants.py#L82

You may change that list if you want to enable a different profile in the built content.

**Reply by ggbecker** [reply]
One more reason why it doesn't exist is that DISA does not publish a STIG for CentOS and that would seem strange to support something that doesn't really exist. Maybe @carlosmmatos can add more about the CentOS situation and the interaction with this project.

**Reply by EngThis** [reply]
Your correct that DISA does not publish a CentOS STIG, however the orginization that approves ATO's for DoD does accept CentOS systems to be STIG'ed using the RHEL STIGs. I currently have a system Approved for use using the RHEL STIG which is why Im asking how I can Build the STIG profile for CentOS to attempt to ease my troubles. Now that CentOS will probably not be used so much in DoD it wouldnt make sense to add the CentOS DISA Profiles into the Builds. Thanks for the help, im going to attempt the to build the CentOS profile like you suggested to see what happens.
